build/AzurePipelinesTemplates/MUX-RunHelixTests-Job.yml

```diff
@@ -13,22 +13,16 @@ parameters:
   # if 'useBuildOutputFromBuildId' is set, we will default to using a build from this pipeline:
   useBuildOutputFromPipeline: $(System.DefinitionId)
   matrix: 
-    Debug_x86:
-      buildPlatform: 'x86'
-      buildConfiguration: 'debug'
-      # %3b is the escape code for ';' which is used as the delimiter
-      openHelixTargetQueues: 'windows.10.amd64.clientrs2.open.xaml%3bwindows.10.amd64.clientrs5.open.xaml'
-      closedHelixTargetQueues: 'windows.10.amd64.clientrs2.xaml%3bwindows.10.amd64.clientrs5.xaml'
     Release_x86:
       buildPlatform: 'x86'
       buildConfiguration: 'release'
-      openHelixTargetQueues: 'windows.10.amd64.client19h1.open.xaml%3bwindows.10.amd64.clientrs3.open.xaml'
-      closedHelixTargetQueues: 'windows.10.amd64.client19h1.xaml%3bwindows.10.amd64.clientrs3.xaml'
+      openHelixTargetQueues: 'windows.10.amd64.clientrs5.open.xaml'
+      closedHelixTargetQueues: 'bwindows.10.amd64.clientrs5.xaml'
```

Comment:
I'm curious: do you know why RS5 is still available but 19H1 is being disabled?

Reply:
RS5 is available as long term servicing, but 19h1 is not.